tests/test_fernet.py

```diff
@@ -156,3 +157,47 @@ def test_no_fernets(self, backend):
     def test_non_iterable_argument(self, backend):
         with pytest.raises(TypeError):
             MultiFernet(None)
+
+    def test_rotate(self, backend):
+        f1 = Fernet(base64.urlsafe_b64encode(b"\x00" * 32), backend=backend)
+        f2 = Fernet(base64.urlsafe_b64encode(b"\x01" * 32), backend=backend)
+
+        mf1 = MultiFernet([f1])
+        mf2 = MultiFernet([f2, f1])
+
+        plaintext = b"abc"
+        mf1_ciphertext = mf1.encrypt(plaintext)
+
+        assert mf2.decrypt(mf1_ciphertext) == plaintext
+
+        rotated = mf2.rotate([mf1.encrypt(plaintext)])[0]
+
+        assert rotated != mf1_ciphertext
```

Comment:
This assert would be true even if the rotation didn't occur because you're calling `mf1.encrypt(plaintext)` again.

Reply:
whoops - I intended to reuse mf1_ciphertext instead of encrypting it again.